src/services/exportInfoMap.ts

```diff
@@ -160,12 +168,10 @@ namespace ts {
             const moduleSymbol = info.moduleSymbol || cachedModuleSymbol || Debug.checkDefined(info.moduleFile
                 ? checker.getMergedSymbol(info.moduleFile.symbol)
                 : checker.tryFindAmbientModule(info.moduleName));
-            const symbolName = exportKind === ExportKind.Default
-                ? InternalSymbolName.Default
-                : info.symbolName;
             const symbol = info.symbol || cachedSymbol || Debug.checkDefined(exportKind === ExportKind.ExportEquals
                 ? checker.resolveExternalModuleSymbol(moduleSymbol)
-                : checker.tryGetMemberInModuleExportsAndProperties(symbolName, moduleSymbol));
+                : checker.tryGetMemberInModuleExportsAndProperties(unescapeLeadingUnderscores(info.symbolTableKey), moduleSymbol),
+                `Could not find symbol '${info.symbolName}' by key '${info.symbolTableKey}' in module ${moduleSymbol.name}`);

```

Comment:
Double checking with @amcasey that this is ok to put in an error message. `info.symbolName` may be a variable name written by a user and `moduleSymbol.name` may be most of a file path. This info will help discover a repro if a user sees it in their TS Server log and can share it.

Reply:
Obviously, the less customer content, the better but, as we've discussed, we have reason to believe this will be broadly useful and the telemetry for the resulting exception will be sanitized appropriately.